src/MagnumPlugins/OpenGexImporter/Test/OpenGexImporterTest.cpp

```diff
@@ -1051,6 +1058,31 @@ void OpenGexImporterTest::extension() {
     }
 }
 
+void OpenGexImporterTest::fileCallbackImage() {
+    if(_manager.loadState("TgaImporter") == PluginManager::LoadState::NotFound)
+        CORRADE_SKIP("TgaImporter plugin not found, cannot test");
+
+    std::unique_ptr<AbstractImporter> importer = _manager.instantiate("OpenGexImporter");
+    CORRADE_VERIFY(importer->features() & AbstractImporter::Feature::FileCallback);
+
+    std::unordered_map<std::string, Containers::Array<char>> files;
+    files["not/a/path/something.ogex"] = Utility::Directory::read(Utility::Directory::join(OPENGEXIMPORTER_TEST_DIR, "texture.ogex"));
+    files["image.tga"] = Utility::Directory::read(Utility::Directory::join(OPENGEXIMPORTER_TEST_DIR, "image.tga"));
```

Comment:
Wait... this should have been `not/a/path/image.tga`, because `image.tga` is next to `something.ogex` and referenced as such. The same in the Assimp importer.

Basically, it's needed to save the path even when calling openFile() with a callback, so the image callbacks get a proper path. This is not a proper path.